releasenotes/notes/deprecate-legacy-code-fe9db8a009b6f5d2.yaml

```diff
@@ -0,0 +1,44 @@
+---
+deprecations:
+  - |
+    Deprecated all code that was migrated to `qiskit_nature/second_q` according to issue:
+    https://github.com/Qiskit/qiskit-nature/issues/749
+
+    The following table provides a rough overview of the old and new package locations.
+    In certain cases, single classes might have been treated differently.
+    Be sure to check out the referenced issues or the migration guide for more details.
```

Comment:
Juts a comment - when we add the migration guide perhaps we can remember to go back and edit the reference here to make it a link. But it all LGTM.